src/main/webapp/WEB-INF/views/css/header.css

```
@@ -0,0 +1,18 @@
+.appLogout {
```

```suggestion
.app-logout {
```
use the same style for all classes